app/helpers/trace_helper.rb

```diff
@@ -2,4 +2,14 @@ module TraceHelper
   def link_to_tag(tag)
     link_to(tag, :tag => tag, :page => nil)
   end
+
+  def trace_image(trace, options = {})
+    options[:class] ||= "trace_image"
+    options[:alt] ||= ""
+
+    link_to(image_tag(trace_icon_path(trace.user, trace),
+                      options.merge(:width => 50, :height => 50)),
+            show_trace_path(trace.user, trace),
+            :class => "d-inline-block")
+  end
```

Comment:
Do we really need a helper for this? If it's supposed to be reusable, why `d-inline-block` is baked into the link?

Why the name `trace_image`? There are two trace images, the name doesn't tell which one this is. Why "image" if it returns a link?

Reply:
I renamed trace_image to trace_preview_image. Not sure if there's a better name for "preview". Then I moved everything which doesn't belong there back to _trace.html.erb. 

The reusable is following the pattern in user_helper.rb. I started adding options[:class], options[:alt], etc in _trace.html.erb, but this looked a bit ugly and cluttered the html code. That's why I moved it to a helper.

Reply:
`trace_icon` sounds fine... changed